rest-api-spec/src/main/resources/rest-api-spec/test/bulk/40_fields.yaml

```diff
@@ -29,10 +29,10 @@
       bulk:
         refresh: true
         body: |
-          { "update": { "_index": "test_index", "_type": "test_type", "_id": "test_id_1", "fields": ["_source"] } }
+          { "update": { "_index": "test_index", "_type": "test_type", "_id": "test_id_1", "_source": true } }
```

Comment:
I think maybe rename the file and the test case? You've changed it so now it is about `_source` instead of `fields`. Maybe add another test case for the filtering?